src/main/java/mk/p/command/Editor.java

```diff
@@ -0,0 +1,75 @@
+package mk.p.command;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class Editor {
+    public JTextArea textField;
+    public String clipboard;
+    private CommandHistory history = new CommandHistory();
+
+    public void init() {
+        JFrame frame = new JFrame("Text editor (type & use buttons, Oleg!)");
+        JPanel content = new JPanel();
+        frame.setContentPane(content);
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        content.setLayout(new BoxLayout(content, BoxLayout.Y_AXIS));
+        textField = new JTextArea();
+        textField.setLineWrap(true);
+        content.add(textField);
+        JPanel buttons = new JPanel(new FlowLayout(FlowLayout.CENTER));
+        JButton ctrlC = new JButton("Ctrl+C");
+        JButton ctrlX = new JButton("Ctrl+X");
+        JButton ctrlV = new JButton("Ctrl+V");
+        JButton ctrlZ = new JButton("Ctrl+Z");
+        Editor editor = this;
+        ctrlC.addActionListener(new ActionListener() {
```

Comment:
In order not to initialize an anonymous class here, you can create a separate class and do the initialization there. That class will extend ActionListener class and accept CommandHistory as a constructor parameter.